ts/components/wallet/SelectionBox.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { TouchableOpacity, View, StyleSheet } from "react-native";
+import { H4 } from "../core/typography/H4";
+import { LabelSmall } from "../core/typography/LabelSmall";
+import { IOColors } from "../core/variables/IOColors";
+
+type Props = {
+  logo?: React.ReactNode;
+  mainText: string;
+  subText: string;
+  ctaText?: string;
+  onPress?: () => void;
+  accessibilityLabel?: string;
+};
+
+const styles = StyleSheet.create({
+  selectionBox: {
+    borderWidth: 1,
+    borderColor: IOColors.bluegreyLight,
+    borderRadius: 8,
+    padding: 16,
+    flexDirection: "row",
+    alignItems: "center"
+  },
+
+  selectionBoxIcon: {
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: "auto",
+    paddingRight: 24
+  },
+
+  selectionBoxContent: {
+    flexGrow: 1,
+    flexShrink: 1,
+    flexBasis: "100%"
+  },
+
+  selectionBoxTrail: {
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: "auto",
+    paddingLeft: 24
+  }
+});
+
+/**
+ * A bordered and rounded box made up by three columns:
+ * - The first one containing an icon
+ * - The second one with a title and a subtitle
+ * - The third one containing a CTA
+ */
+export const SelectionBox = (props: Props) => (
+  <TouchableOpacity
+    disabled={!props.onPress}
+    onPress={props.onPress}
+    accessibilityLabel={props.accessibilityLabel}
+    accessible={props.accessibilityLabel !== undefined}
+  >
+    <View style={styles.selectionBox}>
+      {props.logo && <View style={styles.selectionBoxIcon}>{props.logo}</View>}
```

Comment:
@thisisjp 
I believe the logo should have a min size.
In this test (iPhone12) is just barely visibile


(left figma - right iPhone 12)
![Schermata 2022-03-23 alle 18 31 28](https://user-images.githubusercontent.com/822471/159760912-41ad231f-a188-4cf2-a880-1399fef57518.png)

Reply:
@Undermaken the icon is always on a squared container (24)—the example on the right is misleading because it's a wrong asset, but it has the right size (e.g.: Mastercard)

Reply:
@diegolagospagopa staring from @thisisjp comment we should use this asset
`img/wallet/payment-methods/paypal/paypal_logo.svg`
![Schermata 2022-03-23 alle 18 58 04](https://user-images.githubusercontent.com/822471/159765154-08de3e9c-7fab-4443-9ae7-cde199531764.png)

Reply:
Changed that logo here: https://github.com/pagopa/io-app/pull/3840/commits/59c24aa1aa3021c67321d65ea2cfeafc559621c6# 🙂

Reply:
Btw we are using the other logo in the page used to change the payment method, maybe we should replace it there too?